backend/routers/productRouter.js

```diff
@@ -36,7 +38,13 @@ productRouter.get(
         : order === 'toprated'
         ? { rating: -1 }
         : { _id: -1 };
-
+    const count = await Product.count({
```

Comment:
if nowdays you use Product.count() you will get this issue from nodemon:
DeprecationWarning: collection.count is deprecated, and will be removed in a future version. Use Collection.countDocuments or Collection.estimatedDocumentCount instead

Also, if you are logged as a admin and seller, you will experiment and issue in the productList in the seller's products  view, so, at first, you see only your products as a seller, but once you click in next page, you will redirect to the productList like in admin view, in other words, you will see all products, and in fact, you just have to see your own products as a seller and not all products like in admin view... so, by the moment, with this video, the productList in admin view, and the productList in seller view, are the same thing...

Also, in my opinion, in this video has left, implement this pagination in the home screen... because you set, for example, 4 products for page, and like in homeScreen we haven't implemented the pagination, now, in the homeScreen you only can see 4 products (in this example)

Reply:
I agree with 6969 , i think the teacher have a problem ! Can you give me a way to solve it ?

Reply:
> I agree with 6969 , i think the teacher have a problem ! Can you give me a way to solve it ?

I will remove pagination, since I run into too many issues, which is not worth solving for me, since I will list a few products, but there are indeed issues with pagination that needs to be solved if you want to use the webpage correctly.